.github/workflows/build.yml

```diff
@@ -296,7 +302,7 @@ jobs:
           dotnet nuget push "*.nupkg" --api-key dummy --source PullRequests --skip-duplicate
           
       - name: Push packages (main)
-        if: ${{ github.ref == 'refs/heads/main' && env.HAS_BUILDABLE_COMPONENTS == 'true' }}
+        if: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/rel/')) && env.HAS_BUILDABLE_COMPONENTS == 'true' }}
         run: |
           dotnet nuget add source https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-Labs/nuget/v3/index.json `
```

Comment:
I'm thinking for the rel branch we push to MainLatest, helps us keep track of the signed vs. unsigned packages easier for validation before we push to NuGet and allows more visibility to anyone using the MainLatest feed as a central place to collect all most recent toolkit packages?